### PR TITLE
New event view

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: events/event/views.py [hunk 1]
```diff
@@ -14,31 +14,27 @@
 TZ = get_current_timezone()
 FORMAT = '%b %d %Y %I:%M%p'
 
-EVENT_NOT_EXISTS = JsonResponse({"error_message": "Such event does not exists"}, status=404)
+EVENT_NOT_EXISTS = JsonResponse({"error_messatge": "Such event does not exists"}, status=404)
```
Comment (Scandie) [inline events/event/views.py]: @ForeverYouth fix misstype please

File: events/static/css/events.less [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+.event-card-item{
+	border-bottom: 1px solid#e0e0e0;
+	border-radius: 3px;
+}
+.description-title{
+	font-size: 17px;
+}
+.date{
+	float: left;
+}
```
Comment (Scandie) [inline events/static/css/events.less]: @ForeverYouth add empty line at the end of the file

File: events/static/js/app/events.js [hunk 3]
```diff
@@ -1,6 +1,14 @@
+import Map from './map'
+import {Container, Row, Col} from 'react-grid-system';
+import {Link} from 'react-router';
+
+
+
```
Comment (Scandie) [inline events/static/js/app/events.js]: @ForeverYouth too many empty lines here

File: events/event/views.py (hunk 1 above)
Comment (ghost) [reply events/event/views.py]: +

File: events/static/css/events.less (hunk 2 above)
Comment (ghost) [reply events/static/css/events.less]: Fix

File: events/static/js/app/events.js (hunk 3 above)
Comment (ghost) [reply events/static/js/app/events.js]: +

**Review by yevgenbehen — APPROVED** [review]
+1 from me

**Review by BorysYudin — APPROVED** [review]

**Pull request merged**